customer-commons/src/main/java/no/unit/nva/customer/model/CustomerDao.java

```diff
@@ -66,9 +69,11 @@ public class CustomerDao implements Typed {
     private Sector sector;
     @JsonAlias("rightRetentionStrategy")
     private RightsRetentionStrategyDao rightsRetentionStrategy;
+    private Set<PublicationInstanceTypes> allowFileUploadFor;
 
     public CustomerDao() {
         vocabularies = EMPTY_VALUE_ACCEPTABLE_BY_DYNAMO;
+        allowFileUploadFor = PUBLICATION_INSTANCE_TYPES_EMPTY_VALUE_ACCEPTABLE_BY_DYNAMO;
```

Comment:
Same, ALLOW_FILE_UPLOAD_FOR_EMPTY_VALUE_ACCEPTABLE_BY_DYNAMO is the appropriate name.